pkg/deploy/api/types.go

```diff
@@ -257,6 +257,8 @@ type DeploymentConfig struct {
 	// Details are the reasons for the update to this deployment config.
 	// This could be based on a change made by the user or caused by an automatic trigger
 	Details *DeploymentDetails
+	// Replicas is the total number of pods targeted by this deploymentConfig.
+	Replicas int
```

Comment:
The internal looks pretty weird because when we refactored the model for v1 (from which upstream is derived), we didn't refactor the internal model to match due to time constraints.